client/src/App.tsx

```diff
@@ -23,33 +28,80 @@ function App(): JSX.Element {
               <Switch>
                 <Route exact path="/login" component={Login} />
                 <Route exact path="/signup" component={Signup} />
+
                 <Route exact path="/dashboard">
                   <Dashboard>
                     <Typography component="h5" variant="h5" align="center">
                       Temporary Dashboard Main Content
                     </Typography>
                   </Dashboard>
                 </Route>
+
                 <Route exact path="/profile">
                   <Dashboard>
                     <Profile />
                   </Dashboard>
                 </Route>
+
+                {/* TODO : Refactoring all routes? */}
+                <Route exact path="/profile/edit">
```

Comment:
Consider the usage of path parameters here.

Eg: `<Route to='/profile/:setting' component={Profile}`

Reply:
Applied